
{'date': 'Thu Jul 30 2020 14:59:40.747 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 15
    from _spy.vitollino.main import jogo
ImportError: cannot import name 'jogo'

ImportError
Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 264
    action()
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 15
    from _spy.vitollino.main import jogo
'''},
{'date': 'Thu Jul 30 2020 15:03:04.24 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 15
    from _spy.vitollino.main import vittolino
ImportError: cannot import name 'vittolino'

ImportError
Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 264
    action()
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 15
    from _spy.vitollino.main import vittolino
'''},
{'date': 'Thu Jul 30 2020 15:03:32.337 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 15
    from _spy.vitollino.main import vitollino
ImportError: cannot import name 'vitollino'

ImportError
Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 264
    action()
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 15
    from _spy.vitollino.main import vitollino
'''},
{'date': 'Thu Jul 30 2020 15:33:23.832 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 50
    kwarp()
NameError: name 'kwarp' is not defined
'''},
{'date': 'Thu Jul 30 2020 15:33:44.716 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 50
    Kwarwp()
  module <module> line 32
    self.v = vitollino()
TypeError: 'NoneType' object is not callable
'''},
{'date': 'Thu Jul 30 2020 15:34:07.442 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 15
    from _spy.vitollino.main import vitollino
ImportError: cannot import name 'vitollino'

ImportError
Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 264
    action()
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 15
    from _spy.vitollino.main import vitollino
'''},
{'date': 'Thu Jul 30 2020 15:34:25.234 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 50
    Kwarwp()
  module <module> line 32
    self.v = vitollino()
TypeError: 'NoneType' object is not callable
'''},
{'date': 'Thu Jul 30 2020 15:35:58.811 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 50
    Kwarwp()
  module <module> line 32
    self.v = vitollino()
TypeError: 'NoneType' object is not callable
'''},
{'date': 'Thu Jul 30 2020 15:56:40.823 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 50
    Kwarwp()
  module <module> line 32
    self.v = vitollino()
TypeError: 'NoneType' object is not callable
'''},
{'date': 'Thu Jul 30 2020 16:00:22.544 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 50
    Kwarwp().vai()
  module <module> line 32
    self.v = vitollino()
TypeError: 'NoneType' object is not callable
'''},
{'date': 'Thu Jul 30 2020 16:03:40.422 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 50
  Kwarwp(Jogo).cria(
                   ^
SyntaxError: Unbalanced bracket (
'''},
{'date': 'Thu Jul 30 2020 16:13:56.325 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
Exception: Cannot read property '__class__' of undefined
  module '$exec_2048' line 16
class Kwarwp():
'''},
{'date': 'Thu Jul 30 2020 16:14:20.320 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 54
    Kwarwp(Jogo).cria()
  module <module> line 36
    self.cena = self.cria(cenario=cenario) if vitollino else None
  module <module> line 40
    cena = self.v.c(self.solo)
AttributeError: 'Kwarwp' object has no attribute 'solo'
'''},
{'date': 'Thu Jul 30 2020 16:46:39.411 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 55
  mapa = mapa.split()
  ^
IndentationError: unexpected indent
'''},
{'date': 'Thu Jul 30 2020 16:47:11.637 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 55
  mapa = mapa.split()
  ^
IndentationError: unexpected indent
'''},
{'date': 'Thu Jul 30 2020 16:47:27.965 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 57
  self.lado, self.col = 100, len(mapa[0])
  ^
IndentationError: unexpected indent
'''},
{'date': 'Thu Jul 30 2020 16:47:38.300 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 57
  self.lado, self.col = 100, len(mapa[0])
  ^
IndentationError: unexpected indent
'''},
{'date': 'Thu Jul 30 2020 16:48:22.558 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 57
  self.lado, self.col = 100, len(mapa[0])
  ^
IndentationError: unexpected indent
'''},
{'date': 'Thu Jul 30 2020 16:48:29.515 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 57
  self.lado, self.col = 100, len(mapa[0])
  ^
IndentationError: unexpected indent
'''},
{'date': 'Thu Jul 30 2020 16:48:42.424 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 55
  mapa = mapa.split()
  ^
IndentationError: unexpected indent
'''},
{'date': 'Thu Jul 30 2020 16:54:44.812 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 55
  mapa = mapa.split()
  ^
IndentationError: unexpected indent
'''},
{'date': 'Thu Jul 30 2020 16:57:34.104 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 55
  mapa = mapa.split()
  ^
IndentationError: unexpected indent
'''},
{'date': 'Thu Jul 30 2020 16:58:24.773 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 55
  mapa = mapa.split()
  ^
IndentationError: unexpected indent
'''},
{'date': 'Thu Jul 30 2020 16:59:47.522 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 56
  mapa = mapa.split()
  ^
IndentationError: unexpected indent
'''},
{'date': 'Thu Jul 30 2020 17:00:00.361 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 55
  """Cria um matriz com os elementos descritos em cada linha de texto"""
  ^
IndentationError: unexpected indent
'''},
{'date': 'Thu Jul 30 2020 17:00:18.565 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
Exception: <SyntaxError: Unexpected token )>
'''},
{'date': 'Thu Jul 30 2020 17:03:34.308 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
Exception: <SyntaxError: Unexpected token )>
'''},
{'date': 'Thu Jul 30 2020 17:03:43.729 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
Exception: <SyntaxError: Unexpected token )>
'''},
{'date': 'Thu Jul 30 2020 17:05:14.290 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 72
  return self.v.a(self.GLIFOS[imagem], w=lado, h=lado, x=i*lado, y=j*lado+lado, cena=cena
                                                                                                                                   ^
SyntaxError: invalid syntax
'''},
{'date': 'Thu Jul 30 2020 17:06:53.494 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
Exception: <SyntaxError: Unexpected token )>
'''},
{'date': 'Thu Jul 30 2020 17:07:00.974 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
Exception: <SyntaxError: Unexpected token )>
'''},
{'date': 'Thu Jul 30 2020 17:13:51.102 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
Exception: <SyntaxError: Unexpected token )>
'''},
{'date': 'Thu Jul 30 2020 17:14:37.953 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 74
    Kwarwp(Jogo)
  module <module> line 51
    self.cena = self.cria(mapa=mapa) if vitollino else None
TypeError: cria() got an unexpected keyword argument 'mapa'
'''},
{'date': 'Wed Aug 05 2020 19:24:38.609 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 48
  sol = self.v.a(fabrica["*"].imagem, w=60, h=60, x=0, y=40, cena=cena
                                                                                                 ^
SyntaxError: invalid syntax
'''},
{'date': 'Wed Aug 05 2020 19:25:10.219 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 60
  return self.v.a(imagem, w=lado, h=lado, x=x, y=y, cena=cena
                                                                                                           ^
SyntaxError: invalid syntax
'''},